src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -281,3 +281,15 @@ define void @__quantum__qis__z__ctl(%Array* %.ctls, %Qubit* %.q) {
 }
 
 
+;===============================================================================
+; quantum.qis math functions

```

Comment:
As we discussed, it might make sense have a separate .ll file for math functions, but we can look into this later as it's basically a build structure refactor.

Reply:
#505
(for now I would like to postpone that refactoring/maintenance)